docker-compose.yaml

```diff
@@ -41,6 +41,7 @@ services:
       HOST_NAME: "0.0.0.0"
       SECRET_KEY: CHANGE_ME
       BACKEND_BASE_URI: http://cla-backend:80
+      CLA_ENV: local
```

Comment:
What does this do?
I can see in the cla_public code that the config defaults it to `dev`, and a local kubernetes deployment sets it to `development`
If `local` is just so that it's non-null (for some reason) and not `production` we might be better being consistent.
Do we use the value of `local` for this elsewhere?

Reply:
You're right, according the python code it should be `dev`, `staging` or `prod`.

We should set it to one of these. What do you think of setting it to `dev` for these tests?

For the local  local kubernetes deployment we should probably change it to one the three above, probably `dev`

Reply:
_:bulb: idea_

We could remove complexity if instead of using docker-compose, we provisioned a namespace in Kubernetes where all of our CLA and FALA application coexist.

At that point

- The applications could be destroyed and re-created by `kubectl` commands, removing the need for docker-compose
- We could have the Kubernetes "feature-tests" YAML stored in the application repositories, eliminating duplication and detachment
- We could run the tests against those public URLs any time at any machine, reducing work needed to get it working on CI

We can create a different service for each branch/change: https://medium.com/just-tech/user-acceptance-testing-in-kubernetes-k8s-9cc9ca7363fa

What do you think?

Reply:
I like that _a lot_ tbh

For now I'd say change this to `dev` here, probably change it in `cla_public`'s kubernetes setup if you're at a loose end for a few minutes, and we can start planning to implement this

I also like that branch-specific-hostnames approach for normal app QAing tbh, to remove any mixups about what branch is currently on staging